Library/Red/libred-test.reds

```diff
@@ -0,0 +1,74 @@
+Red/System [
+	Title:   "Red/System Red binding"
+	Author:  "Oldes"
+	File: 	 %red.reds
+	Rights:  "Copyright (C) 2017 David 'Oldes' Oliva. All rights reserved."
+	License: "BSD-3 - https://github.com/red/red/blob/master/BSD-3-License.txt"
+]
+
+#include %red.reds
+
+print-line "libred test started"
+
+my-add: func[
+	[cdecl]
+	"Test libRed routine"
+	a [red_integer!]
+	b [red_integer!]
+	return: [red_integer!]
+	/local ca cb
+][
+	ca: redCInt32 a
+	cb: redCInt32 b
+	redPrint redString "my-add called"
+	redPrint a ;@@ prints: integer   ?!
+	redPrint b ;@@ prints: false     ?!
+	printf ["add called! %ld %ld^/" ca cb] ;@@ this is not visible either:/
```

Comment:
comments are outdated
I can see number being printed on all platforms
`printf` only works on Mac and Linux though